src/NuGet.Core/NuGet.Configuration/Settings/Items/TrustedSignerItem.cs

```diff
@@ -60,7 +60,7 @@ internal TrustedSignerItem(XElement element, SettingsFile origin)
 
             var parsedCertificates = _parsedDescendants.OfType<CertificateItem>().ToList();
 
-            if (parsedCertificates.Count() < 1)
+            if (!parsedCertificates.Any())
```

Comment:
since `parsedCertificates` is a list, I prefer using `Count` property instead of `Any` extension method.

Reply:
``` suggestion
            if (parsedCertificates.Count == 0)
```

Reply:
Done! Change will be reflected in next commit / push.